src/libs/UserUtils.ts

```diff
@@ -115,64 +103,57 @@ function getDefaultAvatarURL(accountID = '', isNewDot = false) {
 
 /**
  * Given a user's avatar path, returns true if user doesn't have an avatar or if URL points to a default avatar
- * @param {String} [avatarURL] - the avatar source from user's personalDetails
- * @returns {Boolean}
+ * @param [avatarURL] - the avatar source from user's personalDetails
  */
-function isDefaultAvatar(avatarURL) {
-    if (
-        _.isString(avatarURL) &&
-        (avatarURL.includes('images/avatars/avatar_') || avatarURL.includes('images/avatars/default-avatar_') || avatarURL.includes('images/avatars/user/default'))
-    ) {
-        return true;
-    }
-
-    // We use a hardcoded "default" Concierge avatar
-    if (_.isString(avatarURL) && (avatarURL === CONST.CONCIERGE_ICON_URL_2021 || avatarURL === CONST.CONCIERGE_ICON_URL)) {
-        return true;
+function isDefaultAvatar(avatarURL: string): boolean {
```

Comment:
I this type `string` correct? if it is, why would we need to check `typeof avatarURL === 'string'` below?

Reply:
Adjusted!